Dropwizard SonarQube properties inherited

We are using 

<parent>
  <groupId>io.dropwizard</groupId>
  <artifactId>dropwizard-dependencies</artifactId>
  <version>2.0.14</version>
</parent>

as specified here: https://www.dropwizard.io/en/latest/manual/upgrade-notes/upgrade-notes-2_0_x.html

The recent changes for https://github.com/dropwizard/dropwizard/pull/3493 have broken our SonarQube Jenkins pipeline the properties specified for sonarqube are getting used in our project.

[INFO] Project key: dropwizard_dropwizard
[INFO] Base dir: /var/lib/jenkins/tmp/workspace/DEV
[INFO] Working dir: /var/lib/jenkins/tmp/workspace/DEV/target/sonar
[INFO] Load project settings for component key: 'dropwizard_dropwizard'
...
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  05:29 min
[INFO] Finished at: 2020-10-27T22:02:00Z
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.sonarsource.scanner.maven:sonar-maven-plugin:3.2:sonar (default-cli) on project xxx: Failed to load the default quality profiles: No organization with key 'dropwizard' ->

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -59,7 +59,7 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      run: ./mvnw --no-transfer-progress -Pjakarta-apis -V -B -ff -s .github/settings.xml org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+      run: ./mvnw --no-transfer-progress -Pjakarta-apis -V -B -ff -s .github/settings.xml -Dsonar.projectKey=dropwizard_dropwizard -Dsonar.organization=dropwizard -Dsonar.host.url=https://sonarcloud.io -Dsonar.moduleKey='${project.artifactId}' org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
       if: matrix.java_version == '8'

--- a/.github/workflows/maven.yml
+++ b/.github/workflows/maven.yml
@@ -59,7 +59,7 @@ jobs:
       env:
         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # Needed to get PR information, if any
         SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
-      run: ./mvnw --no-transfer-progress -Pjakarta-apis -V -B -ff -s .github/settings.xml -Dsonar.projectKey=dropwizard_dropwizard -Dsonar.organization=dropwizard -Dsonar.host.url=https://sonarcloud.io -Dsonar.moduleKey='${project.artifactId}' org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
+      run: ./mvnw --no-transfer-progress -Pjakarta-apis -V -B -ff -s .github/settings.xml -Dsonar.projectKey=dropwizard_dropwizard -Dsonar.organization=dropwizard -Dsonar.host.url=https://sonarcloud.io org.sonarsource.scanner.maven:sonar-maven-plugin:sonar
     - name: Perform CodeQL Analysis
       uses: github/codeql-action/analyze@v1
       if: matrix.java_version == '8'

--- a/pom.xml
+++ b/pom.xml
@@ -80,11 +80,6 @@
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
         <sonar-maven-plugin.version>3.7.0.1746</sonar-maven-plugin.version>
-
-        <sonar.projectKey>dropwizard_dropwizard</sonar.projectKey>
-        <sonar.organization>dropwizard</sonar.organization>
-        <sonar.host.url>https://sonarcloud.io</sonar.host.url>
-        <sonar.moduleKey>${project.artifactId}</sonar.moduleKey>
     </properties>
 
     <developers>

--- a/pom.xml
+++ b/pom.xml
@@ -80,6 +80,8 @@
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <nexus-staging-maven-plugin.version>1.6.8</nexus-staging-maven-plugin.version>
         <sonar-maven-plugin.version>3.7.0.1746</sonar-maven-plugin.version>
+
+        <sonar.moduleKey>${project.groupId}:${project.artifactId}</sonar.moduleKey>
     </properties>
 
     <developers>

